rules/windows/credential_access_lsass_handle_via_malseclogon.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2022/06/29"
+maturity = "production"
+updated_date = "2022/06/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious access to LSASS handle from a call trace pointing to seclogon.dll and with a suspicious access rights
+value, this may indicate an attempt to leak an Lsass handle via abusing the Secondary Logon service in preparation for
+credential access.
```

Comment:
```suggestion
Identifies suspicious access to LSASS handle from a call trace pointing to seclogon.dll and with a suspicious access
rights value. This may indicate an attempt to leak an LSASS handle via abusing the Secondary Logon service in preparation
for credential access.
```